nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -491,35 +483,31 @@ define(
             }
 
             function startTab(tabId) {
-                const selectedTab = controlBarTabs.tabs[tabId];
+                const selectedTab = tabSet.tabs[tabId];
 
-                controlBarTabs.selectedTab = {
+                tabSet.selectedTab = {
                     id: tabId,
                     widget: selectedTab.widget.make({
                         model,
-                        jobId: selectedJobId,
                     }),
                 };
                 return _startTab(tabId, { model });
             }
 
             function _startTab(tabId, startArgs) {
-                ui.activateButton(controlBarTabs.selectedTab.id);
+                ui.activateButton(tabSet.selectedTab.id);
 
                 const node = document.createElement('div');
                 node.classList.add(`kb-app-cell-tab-pane__container--${tabId}`, 'clearfix');
                 ui.getElement('body.widget.tab-pane.widget').appendChild(node);
                 startArgs.node = node;
                 startArgs.bus = runtime.bus();
-                return controlBarTabs.selectedTab.widget.start(startArgs);
+                return tabSet.selectedTab.widget.start(startArgs);
             }
 
             function stopTab() {
-                ui.deactivateButton(controlBarTabs.selectedTab.id);
-                if (controlBarTabs.selectedTab.widget.getSelectedJobId) {
-                    selectedJobId = controlBarTabs.selectedTab.widget.getSelectedJobId();
-                }
```

Comment:
`getSelectedJobId()` is only referenced in this file, and the `selectedJobId` / `jobId`is not used by any of the widgets ==> :wave: